Scripts/MC/MaxGrapplingRange.gd

```diff
@@ -7,8 +7,8 @@ var aiming
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
-	aim = preload("res://Assets/Sprites/Aim.png")
-	aiming = preload("res://Assets/Sprites/Aiming.png")
+	aim = preload("res://Assets/Sprites/Player/Aim.png")
```

Comment:
Can this path be added to GlobalPaths.gd and then just referenced here as 
aim = preload(GlobalPaths.PLAYER_AIM_PATH)
?